dandiapi/api/views/serializers.py

```diff
@@ -173,7 +173,7 @@ class Meta(DandisetSerializer.Meta):
 class DandisetQueryParameterSerializer(serializers.Serializer):
     draft = serializers.BooleanField(default=True)
     empty = serializers.BooleanField(default=True)
-    embargoed = serializers.BooleanField(default=True)
+    embargoed = serializers.BooleanField(default=False)
```

Comment:
I think it might be insufficient to retain current webui behavior, since when authenticated we do see embargoed ones "by default". May be there should be need to implement handling of default None, which would maintain current behavior

Reply:
I'll be including the appropriate webui changes in this PR as well, so that everything is consistent.